plutus-contract/test/Spec/Contract/TxConstraints.hs

```diff
@@ -0,0 +1,259 @@
+{-# LANGUAGE DataKinds             #-}
```

Comment:
@james-iohk Feel free to verify that the testing is valid and propose new testing for this constraint.